cli/flox/src/commands/containerize/macos_containerize_proxy.rs

```diff
@@ -102,11 +107,27 @@ impl ContainerBuilder for ContainerizeProxy {
             FLOX_FLAKE,
             // Use a more specific commit if available, e.g. pushed to GitHub.
             // TODO: Doesn't always work: https://github.com/flox/flox/issues/2502
-            flox_version.commit_sha().unwrap_or(flox_version_tag)
+            (*FLOX_CONTAINERIZE_FLAKE_REF_OR_REV)
+                .clone()
+                .unwrap_or(flox_version.commit_sha().unwrap_or(flox_version_tag))
         );
         command.args(["run", &flox_flake, "--"]);
 
         // Inception L3: Flox args.
+
+        // TODO: this should probably be a method on Verbosity
+        match flox.verbosity {
```

Comment:
(non-blocking, note) We can also use this for `nix` args in the follow-up ticket:

- https://github.com/flox/flox/issues/2498